build.gradle

```diff
@@ -0,0 +1,101 @@
+plugins {
+    id 'jacoco'
+    id 'groovy'
+}
+
+group "org.ods"
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = 1.8
+
+ext {
+    nexus_url = project.findProperty('nexus_url') ?: System.getenv('NEXUS_HOST')
+    nexus_user = project.findProperty('nexus_user') ?: System.getenv('NEXUS_USERNAME')
+    nexus_pw = project.findProperty('nexus_pw') ?: System.getenv('NEXUS_PASSWORD')
+    no_nexus = (project.findProperty('no_nexus') ?: System.getenv('NO_NEXUS') ?: false).toBoolean()
+}
+
+repositories {
+    if (no_nexus) {
+        println("using repositories 'jcenter' and 'mavenCentral'")
+        jcenter()
+        mavenCentral()
+
+        maven {
+            url "https://repo1.maven.org/maven2/"
+            mavenContent {
+                releasesOnly()
+            }
+        }
+
+        maven {
+            url "https://repo.jenkins-ci.org/releases"
+            mavenContent {
+                releasesOnly()
+            }
+        }
+    } else {
+        println("using nexus repositories")
+        if (!nexus_url) {
+            throw new GradleException('Nexus URL not specified!')
+        }
+        maven() {
+            url "${nexus_url}/repository/jcenter/"
+            credentials() {
+                username = "${nexus_user}"
+                password = "${nexus_pw}"
+            }
+        }
+        maven() {
+            url "${nexus_url}/repository/maven-public/"
+            credentials() {
+                username = "${nexus_user}"
+                password = "${nexus_pw}"
+            }
+        }
+        maven() {
+            url "${nexus_url}/repository/atlassian_public/"
+            credentials {
+                username = "${nexus_user}"
+                password = "${nexus_pw}"
+            }
+        }
+    }
+}
+
+dependencies {
+    implementation "org.codehaus.groovy:groovy-all:2.4.12"
```

Comment:
@oalyman any specific reasons you're not using the same dependencies, we use in the MRO shared library?

Reply:
2.4.12 is the groovy version jenkins supports.

Reply:
@oalyman good finding! Do you want to provide a quick PR for the MRO?

Reply:
Sure. I'll do it in a second